addons/hr_attendance/views/hr_attendance_view.xml

```diff
@@ -69,7 +69,8 @@
         <field name="arch" type="xml">
             <form string="Employee attendances" duplicate="false">
                 <header>
-                    <field name="overtime_status" widget="statusbar" readonly="0" options="{'clickable': '1'}"/>
+                    <field name="overtime_status" widget="statusbar" invisible="validated_overtime_hours == 0" readonly="0" options="{'clickable': '1'}"/>
```

Comment:
As ``validated_overtime_hours`` is a float field, this could be wrong in the case the value is 0.0000000001 or -0.00000001.

I suggest you add a non-stored computed field ``no_validated_overtime_hours`` using ``float_compare`` properly to decided whether the amount is equal to 0 or not.